### DISCUSSION TITLE
Only update price & stock

### DISCUSSION THREAD

**Discussion by lukasbrisman** [datafeedr/wordpress-plugins#36] [Help]
Hey guys!

I'm going to alter all the data on the product after the first import, so i would need help in setting up the updates so ONLY the price (and sale price) and the stock level (delete if not in stock or deleted from merchant) are updated except for when they are first imported. 

My thought is to import everything the first time, write descriptions (with the help of AI), fix the categories and attributes and then have datafeedr api update the price and stock levels. 

How can i do this?! :)

Thanks!

Lukas

**Comment by EricBusch** [comment]
Hi

Here's how you can do this: 

- https://datafeedrapi.helpscoutdocs.com/article/50-prevent-product-titles-descriptions-excerpts-from-being-overwritten-during-product-set-updates
- https://datafeedrapi.helpscoutdocs.com/article/218-prevent-categories-from-being-overwritten-during-a-product-set-update
- https://datafeedrapi.helpscoutdocs.com/article/44-prevent-attributes-from-being-overwritten-during-product-set-updates

Hope this helps!

Eric
